### PR TITLE
Portfolio merge

### PR DESCRIPTION
**Pull request by Thinus01**
Updated HTML & CSS with middle part and about me
Added Linters
Added README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+            <li id="languages">
+
+                <section id="langbox">
+                    <p id="lang_text">Languages</p>
+                    <img id="down_arrow" src="images/downarrow.svg">
+                </section>
+                <ul>
+                    <li><a id="langicon" href="#"><img src="images/javascript.svg" alt="javascript"></a></li>
+                    <li><a id="langicon" href="#"><img src="images/html.svg" alt="html"></a></li>
+                    <li><a id="langicon" href="#"><img src="images/css.svg" alt="css"></a></li>
```
Comment (Debas-31) [inline index.html]: [ ] Great at implementing the about-myself section design, but some issues to figure out in your design. As much as possible, stick to the Figma template you choose. Make sure of the alignment of the section, like the CSS property margin and padding as the designed template.

| Figma template design  | Your design|
| ------------- | ------------- |
|Screenshot from 2022-10-29 16-42-58| Screenshot from 2022-10-29 16-47-10|

File: index.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+                        <p id="tjavascript">javascript</p>
+                    </section>
+                </section>
+                <div id="action">
+                    <button id="normal_button">
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
```
Comment (Debas-31) [inline index.html]: - [ ] Please note that all your cards should be in one section. You have used so many sections in a section that it's better to use your cards, which can be articles in a section or just divs in a section. Also, you are to align the contents of the section (that is, the cards) using a CSS grid as demanded by project requirements. Kindly fix this, please.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio(3).svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Multi-Post Stories</p>
+                    <section id="frame2">
```
Comment (Debas-31) [inline index.html]: - [ ] Please note that you should have only one h1 in the whole of your document; kindly change this to any other heading tag(from h2 to h5). It is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+                    </button>
+                </div>
+            </section>
+        </section>
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio(2).svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Tonic</p>
+                    <section id="frame2">
```
Comment (Debas-31) [inline index.html]: - [ ] Please note that you should have only one h1 in the whole of your document; kindly change this to any other heading tag(from h2 to h5). It is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio(1).svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Multi-Post Stories</p>
```
Comment (Debas-31) [inline index.html]: - [ ] Please note that you should have only one h1 in the whole of your document; kindly change this to any other heading tag(from h2 to h5). It is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+        </ul>
+    </section>
+
+    <section id="works">
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio.svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Tonic</p>
+                    <section id="frame2">
```
Comment (Debas-31) [inline index.html]: - [ ] Please note that you should have only one h1 in the whole of your document; kindly change this to any other heading tag(from h2 to h5). It is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

**Review by Debas-31 — CHANGES_REQUESTED** [review]
## Hi @Thinus01  :wave:
# Status: Required Change :recycle:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- Commit message :heavy_check_mark:  
 
## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+                    <button id="normal_button">
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio(1).svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
```
Comment (OyePriscilla) [inline index.html]: - [ ] Please kindly effect the changes that the previous reviewer requested. 😄.

> Please note that all your cards should be in one section. You have used so many sections in a section that it's better to use your cards, which can be articles in a section or just divs in a section. Also, you are to align the contents of the section (that is, the cards) using a CSS grid as demanded by project requirements. Kindly fix this, please.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio(2).svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Tonic</p>
```
Comment (OyePriscilla) [inline index.html]: - [ ] You've not implemented this as well for all the project titles. Be encouraged, you can do this. 👍 You can change the `p `tag to `h3 `tag. 😄 

> Please note that you should have only one h1 in the whole of your document; kindly change this to any other heading tag(from h2 to h5). It is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+            <li id="languages">
+
+                <section id="langbox">
+                    <p id="lang_text">Languages</p>
+                    <img id="down_arrow" src="images/downarrow.svg">
+                </section>
+                <ul>
+                    <li><a id="langicon" href="#"><img src="images/javascript.svg" alt="javascript"></a></li>
+                    <li><a id="langicon" href="#"><img src="images/html.svg" alt="html"></a></li>
+                    <li><a id="langicon" href="#"><img src="images/css.svg" alt="css"></a></li>
```
Comment (OyePriscilla) [inline index.html:245]: - [ ] Kindly make a little adjustment here as well. 👍 

## Your design

crr22


## Figma design 

crr24

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Thinus01 😄 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
* Linters setup ✔️
* GitHub flow ✔️
* Commit message ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+                    <section id="tag1">
+                        <p id="thtml">html</p>
+                    </section>
+                    <section id="tag2">
+                        <p id="tcss">css</p>
+                    </section>
+                    <section id="tag3">
+                        <p id="tjavascript">javascript</p>
+                    </section>
+                </section>
```
Comment (OyePriscilla) [inline index.html]: - [ ] According to best practices, you can only use `section` tag to group related elements. The format below might be helpful for you.

```
<section>
                <div>
                    <img src="resources/img/SnapshootPortfolio.png" alt="snapshoot">
                </div>
                <div>
                    <div>
                        <h1 class="tonic">Tonic</h1>
                        <ul>
                            <li><a href="#">CANOPY</a></li>
                            <li><a href="#">Back End Dev</a></li>
                            <li><a href="#">2015</a></li>
                        </ul>
                    </div>
                    <p>Proof read my works here, no account sign-ups required</p>
                    <ul>
                        <li>html</li>
                        <li>css</li>
                        <li>javascript</li>
                    </ul>
```

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
# Hi @Thinus01 👋
## Status: Required Change ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5754646735474252343558364e6278714a4b2f67697068792e676966

## Highlights
Linters setup ✔️
GitHub flow ✔️
Commit message ✔️

## Required Changes ♻️

Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Lameck1 — COMMENTED** [review]
Hello @Thinus01 👋,

_This is an additional review as a result of the second opinion request._

The concerns the reviewer has raised are genuine. It's not just a matter of best practices, it is about using HTML tags for what they are meant. The reviewer has explained the concerns and why the change is necessary and has even given you an example to work with. You cannot use the `<section>` tag to create a list, we have the `<ul>` or the `<ul>` for that. You cannot also use the `<section>` tag to wrap every HTML with it.

Every HTML tag has meaning and this meaning is interpreted by our web browser to lay out the information properly. We should strive to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). It is also the work of the Code Reviewers to check whether we are writing semantic HTML. Please, don't hold it against @OyePriscilla for giving you valuable feedback, that is what is expected of the Code Reviewers.

Would it make you happy if your projects were approved with gross mistakes? Have you thought about how that will affect your chances of getting a job? If we don't get exposed to the right way of doing things in the code review process, how can we learn?

Please work on the required changes and submit them for another review. Consider working with the example that you've been provided in the last review.

Keep rocking 🚀!

File: index.html [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+                        <img id="counter">
+                        <div id="role">
+                            <h3 id="pcareer">Back End Dev</h3>
+                        </div>
+                        <img id="counter">
+                        <div id="year_label">
+                            <h5 id="pcareer">2015</h5>
+                        </div>
+                    </div>
+                </div>
```
Comment (ichala) [inline index.html]: - [ ] Great job removing the sections tags from the lists 👏  , but as mentioned by both  the previous reviewer and the Sr Reviewer, we should be semantically correct and use the tags `<ul>` & `<li>` for lists, here's a hint on how to do it : 

❌ Incorrect : 
```HTML
       <div id="frame2">
                        <div id="client">
                            <h2 id="pcanopy">CANOPY</h2>
                        </div>
                        <img id="counter">
                        <div id="role">
                            <h3 id="pcareer">Back End Dev</h3>
                        </div>
                        <img id="counter">
                        <div id="year_label">
                            <h5 id="pcareer">2015</h5>
                        </div>
             </div>
```

✔️  Correct : 

```HTML
       <ul id="frame2">
                        <li id="client">
                            <h2 id="pcanopy">CANOPY</h2>
                        </li>
                        <li id="role">
                            <img id="counter">
                            <h3 id="pcareer">Back End Dev</h3>
                        </li>
                        <li id="year_label">
                            <img id="counter">
                            <h5 id="pcareer">2015</h5>
                        </li>
          </ul>
```

  _**🛑 Kindly fix it for all of your cards 🙏**_

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Thinus01 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Removed sections tags✔️ 
 
 ## Required Changes ♻️

- [ ] 🛑 **_Check The comments below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fherrerao — APPROVED** [review]
# ✅  🎆  STATUS APPROVED! ✅ 🎆 
#
Hi @Thinus01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great Job Yes GIF by Hollywood Suite


## Highlights
✔️  No linter errors.
✔️  You used the correct GitHub-Flow.
✔️  Good README
✔️  Good Commit Messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**